### PR TITLE
Add support for Scala 2.12.11

### PR DESCRIPTION
**Pull request by sideeffffect**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,13 +4,15 @@ scala:
...
   - 2.12.10
+  - 2.12.11
   - 2.13.0
   - 2.13.1
 jdk:
   - openjdk8
   - openjdk12
 cache:
   directories:
+  - $HOME/.cache/coursier
```
Comment (cb372) [inline .travis.yml:15]: 👍

**Review by cb372 — APPROVED** [review]
Thanks! I'll make a release asap.

**Comment by kubukoz** [comment]
Hey, how's that release going? :D

**Comment by cb372** [comment]
@sideeffffect @kubukoz Done! 😅 

I need to set up sbt-ci-release on this repo...

**Pull request merged**